core/Symbols.cc

```diff
@@ -1171,6 +1171,57 @@ vector<Symbol::RequiredAncestor> Symbol::requiredAncestors(const GlobalState &gs
     return res;
 }
 
+// All required ancestors by this class or module
+vector<Symbol::RequiredAncestor> requiredAncestorsTransitiveInternal(const GlobalState &gs, SymbolRef sym,
+                                                                     std::vector<SymbolRef> seen) {
+    std::vector<Symbol::RequiredAncestor> res;
+
+    if (std::find(seen.begin(), seen.end(), sym) != seen.end()) {
```

Comment:
Could you use `absl::c_find(seen, sym)` instead? It makes it easier to avoid common problems with iterators.